sample/src/main/java/com/karumi/dexter/sample/SampleActivity.java

```diff
@@ -144,11 +145,6 @@ private void createPermissionListeners() {
                 R.string.all_permissions_denied_feedback)
                 .withOpenSettingsButton(R.string.permission_rationale_settings_button_text)
                 .build());
-    cameraPermissionListener = new CompositePermissionListener(feedbackViewPermissionListener,
-        SnackbarOnDeniedPermissionListener.Builder.with(rootView,
-            R.string.camera_permission_denied_feedback)
-            .withOpenSettingsButton(R.string.permission_rationale_settings_button_text)
-            .build());
```

Comment:
@pedrovgs ok, that's fine. The only issue here is that it was using the `SnackbarOnDeniedPermissionListener` and it does not support being shown from a background thread. Should I add this support to the listener?

Same happens with `DialogOnDeniedPermissionListener`.

I could check the running thread inside `onPermissionDenied` in the listener and if it's a background thread using a handler to post the message to the UI thread